### PR TITLE
updated checklist for the july virtual meeting.

### PR DESCRIPTION
**Pull request by humnaawan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,276 +2,313 @@
...
-      text-decoration: line-through;
-      color: #999;
-    }
+     <title>Checklist for the LSST DESC July Virtual Meeting</title>
+     <meta http-equiv="content-type" content="text/html; charset=utf-8">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" rel="stylesheet">
+     <style>
+     html {
+          font-family: sans-serif, Helvetica, Arial;
```
Comment (yymao) [inline index.html]: `sans-serif` should be placed as the last item as a fallback option. If you prefer a different font, you can add the specific font name as the first item.

```suggestion
          font-family: Helvetica, Arial, sans-serif;
```

File: index.html [hunk 2]
```diff
@@ -2,276 +2,313 @@
...
+          <p>This site uses cookies to keep track your checklist progress. The data are stored in your browser locally only.</p>
+          <p>If you have questions or suggestions about this page, please
+               <a href="https://github.com/LSSTDESC/pre-meeting-checklist/issues" title="link to GitHub">open an issue</a> or contact
+               <a href="https://yymao.github.io">Yao-Yuan Mao</a> or <a href="mailto:agkim@lbl.gov?Subject=DESC%20Meeting%20Checklist" target="_top">Alex Kim</a>.</p>
+     </div>
+
+     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+     <script src="https://cdn.jsdelivr.net/npm/js-cookie@2/src/js.cookie.min.js"></script>
+     <script>
+     const cookie_key = "lsst_desc_pre_meeting_checklist_arizona2020";
```
Comment (yymao) [inline index.html]: Update cookie key
```suggestion
     const cookie_key = "lsst_desc_pre_meeting_checklist_july2020";
```

File: index.html [hunk 3]
```diff
@@ -2,276 +2,313 @@
...
+          margin: 30px 0;
+          position: sticky;
+          top: -1px;
+     }
+
+     #progress {
+          width: 1%;
+          height: 100%;
+          float: left;
+          background-color: #66B66C;
```
Comment (yymao) [inline index.html:49]: Any reason not to use the same color scheme as the links and checkboxes?

File: index.html [hunk 4]
```diff
@@ -2,276 +2,313 @@
...
+     }
 
-    .footer {
-      text-align: center;
-      margin-top: 72px;
-    }
-  </style>
+     @supports not (zoom:1.5) {
+          input[type=checkbox] {
+               transform: scale(1.6);
```
Comment (yymao) [inline index.html]: The change from zoom 2 -> 1.5 make the checkbox not aligned with the text. What's the motivation for this change?

File: index.html [hunk 5]
```diff
@@ -2,276 +2,313 @@
...
+               hashtag when reporting highlights from the meeting! Remember the Code of Conduct mentioned above!
+          </div>
+     </div>
+     <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+     <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+     <div class="footer">
+          <p>This site uses cookies to keep track your checklist progress. The data are stored in your browser locally only.</p>
+          <p>If you have questions or suggestions about this page, please
+               <a href="https://github.com/LSSTDESC/pre-meeting-checklist/issues" title="link to GitHub">open an issue</a> or contact
+               <a href="https://yymao.github.io">Yao-Yuan Mao</a> or <a href="mailto:agkim@lbl.gov?Subject=DESC%20Meeting%20Checklist" target="_top">Alex Kim</a>.</p>
```
Comment (yymao) [inline index.html:275]: Change the contacts here?

**Review by yymao — CHANGES_REQUESTED** [review]
@humnaawan I've reviewed the webpage design, but not yet the content.

**Comment by yymao** [comment]
Content looks fine to me. Though I'm not on the SOC/LOC so someone else should look at it.

File: index.html (hunk 3 above)
Comment (humnaawan) [reply index.html:49]: i found the blue to be confusing - and honestly didn't realize that what i presumed to be a header for a progress bar. i tried to add a progress label (e.g. 5%, 50%, 100%) but i didnt have enough time to modify the code.

File: index.html (hunk 4 above)
Comment (humnaawan) [reply index.html]: oh the checkboxes were appearing very blurry on my screen and changing the zoom in scale fixed the issue.

File: index.html (hunk 5 above)
Comment (humnaawan) [reply index.html:275]: you and alex are the ones with admin privileges to the repo so i'm not sure if it makes sense to change the contact info for "questions or suggestions about **this** page".

**Comment by humnaawan** [comment]
also, i issued the PR only after a green signal from the SOC. perhaps @katrinheitmann @burchat can comment.

**Comment by katrinheitmann** [comment]
The SOC approves the list

**Review by yymao — APPROVED** [review]
@humnaawan I've fixed the blurred checkbox issue and add a number to the progress bar. Can you verify that both work as expected?  Thanks.

**Comment by humnaawan** [comment]
perfect - thank you!
